docs/samples/Microsoft.ML.Samples/Dynamic/Calibrator.cs

```diff
@@ -34,8 +34,8 @@ public static void Example()
                 HasHeader = true,
                 Columns = new[]
                     {
-                        new TextLoader.Column("Sentiment", DataKind.BL, 0),
-                        new TextLoader.Column("SentimentText", DataKind.Text, 1)
+                        new TextLoader.Column("Sentiment", typeof(bool), 0),
```

Comment:
In issue #2006, I had [a comment](https://github.com/dotnet/machinelearning/issues/2006#issuecomment-451511350). You will note that you are picking the option that I argued was the worst of all possible worlds. It offers absolutely no hints to the user of what might be an acceptable type. How does a user know that a `bool` is acceptable, but, say, a `List<bool>` is not>? This is much, *much* worse than what it is supposedly meant to replace.

Reply:
Ok. I will make `ColumnInfo` strongly-typed based on the desired output type.

---
In reply to: [258657461](https://github.com/dotnet/machinelearning/pull/2661#discussion_r258657461) [](ancestors = 258657461)

Reply:
That is one possibility. We should also remain open to the possibility that `DataKind` will persist, just in a changed form. So while doing this work, think about whether it is making the usage more or less clear.

It is a perfectly acceptable outcome to say, "you know what, this enumeration is still useful, just in a vastly abbreviated scope," if we can make that argument. It is unclear to me which is correct. I would not start with the assumption that datakind needs to be internalized and destroyed. (W.r.t. its prior role in `ColumnType`, absolutely!!)

Reply:
How about this one for a new form of `DataKind`?
```csharp
    public static class ScalarTypes
    {
        public static Type Float => typeof(float);

        public static Type Double => typeof(double);
    }
```

---
In reply to: [258755422](https://github.com/dotnet/machinelearning/pull/2661#discussion_r258755422) [](ancestors = 258755422)

Reply:
Discussed offline. We decided internalizing DataKind and creating an simplified alternative.

---
In reply to: [259025888](https://github.com/dotnet/machinelearning/pull/2661#discussion_r259025888) [](ancestors = 259025888,258755422)